src/menu/index.tsx

```diff
@@ -21,6 +21,7 @@ const Menu = ({ className, items }: MenuProps) => (
         rightAddon={item.rightAddon}
         key={item.id}
         onClick={item.onClick}
+        status={item.status}
```

Comment:
What if the status is undefined?

Reply:
The item choice status default to 'default' like before.